holoviews/element/chart.py

```diff
@@ -56,10 +56,27 @@ def __getitem__(self, index):
 
 class Scatter(Selection2DExpr, Chart):
     """
-    Scatter is a Chart element representing a set of points in a 1D
-    coordinate system where the key dimension maps to the points
-    location along the x-axis while the first value dimension
-    represents the location of the point along the y-axis.
+    The `Scatter` element visualizes as markers placed in a space of one
+    independent variable, traditionally denoted as `x`, against a dependent
+    variable, traditionally denoted as `y`.
+
+    Scatter elements most naturally overlay with other elements that express
+    dependent relationships between the x and y axes in two-dimensional space,
+    such as Chart types like `Curve`.
+    
+    Reference: [Bokeh](http://holoviews.org/reference/elements/bokeh/Scatter.html), [Matplotlib](http://holoviews.org/reference/elements/matplotlib/Scatter.html), [Plotly](http://holoviews.org/reference/elements/plotly/Scatter.html)
```

Comment:
Markdown won't work sadly.

Reply:
What link formatting _will_ work? I'd be happy to see this merged once the links are formatted in a usable way.

Reply:
I think markdown is fine for vs code tooltips but for ref API I think it needs to be reST